JsonRocket.Test/ExtractorTests.cs

```diff
@@ -65,5 +65,70 @@ public void SkipsIntermediateArrays()
             actual[1].Path.Should().Be("Value.Key");
             actual[1].Value.ReadInt32().Should().Be(987);
         }
+
+        [Fact]
+        public void ArrayIndexingSample()
+        {
+            var trie = new Trie();
+            trie.Add("ExecutionInstructions[1].Code");
+
+            var tokenizer = new Tokenizer();
+            tokenizer.Reset(@"{""Key"":""17081100021800"",""ExecutionInstructions"":[{""Code"":""Manager"",""Value"":""FPA""},{""Code"":""StartTime"",""Value"":""09:50""}]}".ToInput());
```

Comment:
the tokenizer supports single quotes for fields/strings ... let's use those to make the inline json more readable